Ginger/GingerCoreNET/Run/GingerRemoteExecutionUtils.cs

```diff
@@ -35,15 +35,15 @@ public List<RunSetReport> GetSolutionRunsetsExecutionInfo(Guid soluionGuid)
         {
             var runSetReports = new List<RunSetReport>();
             var baseURI = GetReportDataServiceUrl();
-            if (!string.IsNullOrEmpty(baseURI))
+            if (!string.IsNullOrEmpty(baseURI) && WorkSpace.Instance.Solution.LoggerConfigurations.PublishLogToCentralDB == ExecutionLoggerConfiguration.ePublishToCentralDB.Yes)
             {
                 using (var httpClient = new HttpClient())
                 {
                     var endpoint = baseURI + "api/AccountReport/GetRunsetsExecutionInfoBySolutionID/" + soluionGuid;
                     var response = httpClient.GetAsync(endpoint).Result;
                     if (!response.IsSuccessStatusCode)
                     {
-                        Reporter.ToLog(eLogLevel.ERROR, "Error occurred during GetSolutionRunsetsExecutionInfo() :" + response.Content.ReadAsStringAsync().Result.ToString());
+                        Reporter.ToLog(eLogLevel.INFO, "Error occurred during GetSolutionRunsetsExecutionInfo() :" + response.Content.ReadAsStringAsync().Result.ToString());
```

Comment:
Add info logs only in case of Not Found response code. for other errors, we still need to log error

Reply:
Resolved